electron/ironfish/IronFishManager.ts

```diff
@@ -159,6 +161,47 @@ export class IronFishManager implements IIronfishManager {
     await walletDb.db.close()
   }
 
+  // used to reset node if datadir exists but is incompatible with mainnet
+  async resetNode(): Promise<void> {
+    log.log(
+      '----------- resetting chain due to network incompatibility ----------------'
+    )
+    await this.node.shutdown()
+    await this.node.closeDB()
+    const hostFilePath: string = this.sdk.config.files.join(
+      this.sdk.config.dataDir,
+      HOST_FILE_NAME
+    )
+    await fsAsync.rm(hostFilePath, { recursive: true, force: true })
```

Comment:
question: Does this delete the whole data dir or just the db?

Reply:
It only removes `hosts.json`